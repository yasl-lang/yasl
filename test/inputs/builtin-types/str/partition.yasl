

io.setformat('r')

echo 'abc::def'->partition('::')
echo 'abc de'->partition(' ')
echo ':abc:def'->partition(':')
echo ':abc:def;'->partition(';')
echo 'abc def'->partition('z')

