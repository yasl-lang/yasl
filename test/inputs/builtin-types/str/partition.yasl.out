'abc', 'def'
'abc', 'de'
'', 'abc:def'
':abc:def', ''
undef, undef
